src/cs50/cs50.py

```diff
@@ -11,6 +12,13 @@
 from traceback import format_exception
 
 
+# Configure default logging handler and formatter
+logging.basicConfig(format="%(levelname)s: %(message)s", level=logging.DEBUG)
+
+# Patch formatException
+logging.root.handlers[0].formatter.formatException = lambda exc_info: _formatException(*exc_info)
```

Comment:
Is `[0]` guaranteed to exist?

Reply:
It should but added `try/except` just to be safe.